File: DigitalDisplay.WEB/src/models/advertisement.ts
export interface Advertisement {
  id?: number;
  name: string;
  categoryId: number;
  advertisementTypeId: number;
  graphicalContent?: any;
  graphicalContentFilename?: string;
  text?: string;
  textFontFamily?: string;
  textFontSize?: number;
  textFontColor?: string;
  textVerticalAlignment?: number;
  textHorizontalAlignment?: number;
  textScrolling?: number;
  textPadding?: string;
  imageFit?: number;
  createdDate: Date;
  createdBy?: string;
  updatedDate?: Date;
  advertisementType?: string;
  categoryName?: string;
  archive?: boolean;
  birthdayData?: Object;
}

File: DigitalDisplayBO.API/Models/Advertisement.cs
using System;
using System.Collections.Generic;

namespace DigitalDisplayBO.API.Models
{
    public partial class Advertisement
    {
        public Advertisement()
        {
            PanelAdvertisements = new HashSet<PanelAdvertisement>();
        }

        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public int CategoryId { get; set; }
        public int AdvertisementTypeId { get; set; }
        public byte[]? GraphicalContent { get; set; }
        public string? GraphicalContentFilename { get; set; }
        public short? ImageFit { get; set; }
        public string? Text { get; set; }
        public string? TextFontFamily { get; set; }
        public int? TextFontSize { get; set; }
        public string? TextFontColor { get; set; }
        public short? TextVerticalAlignment { get; set; }
        public short? TextHorizontalAlignment { get; set; }
        public string? TextPadding { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string? CreatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public string? UpdatedBy { get; set; }
        public short? TextScrolling { get; set; }
        public bool? Archive { get; set; }
        public string? BirthdayData { get; set; }

        public virtual AdvertisementType AdvertisementType { get; set; } = null!;
        public virtual Category Category { get; set; } = null!;
        public virtual ICollection<PanelAdvertisement> PanelAdvertisements { get; set; }
    }
}
